### ISSUE TITLE
Weekly scheduled exports are listed with the wrong day

### ISSUE THREAD

**Issue by rivaldi8** [codinguser/gnucash-android#617]
#### Steps to reproduce the behaviour
Create a weekly scheduled export on Sundays.

#### Expected behaviour
The scheduled export is listed as "Weekly on Sunday".

#### Actual behaviour
The scheduled export is listed as "Weekly on Thursday".

Reported through UserVoice (ticket 143).

**Comment by rivaldi8** [comment]
Confirmed. It seems it always shows "Thursday" independently of the day chosen.

**Comment by codinguser** [comment]
hmm, that's a weird one. Will look at the parsing again when I have some time to see if it is coming from our code or a library

**Comment by rivaldi8** [comment]
I already found the problem. Sorry, I should've commented on it earlier.

[Recurrence.getRepeatString](https://github.com/rivaldi8/gnucash-android/blob/6032f0541cca33f34f0c594fedaa7222751b2cf7/app/src/main/java/org/gnucash/android/model/Recurrence.java#L137-L137) is taking the day of the week from `mPeridoStart` instead of from `mByDay`. So "Thursday" is just a coincidence, it's really showing the day in which the scheduled action was created. Then is when I noticed that we are not even using the day of the week set by the user when scheduling actions, so I preferred to look at the hole thing later.
